lxd/events/common.go

```diff
@@ -35,68 +32,10 @@ type listenerCommon struct {
 }
 
 func (e *listenerCommon) heartbeat() {
```

Comment:
Shall we rename this to `start()` as that would be more relevant for all implementations now.